hnn_core/batch_simulate.py

```diff
@@ -70,10 +73,22 @@ def __init__(self, set_params, net_name='jones', tstop=170,
         summary_func : callable, optional
             A function to calculate summary statistics from the simulation
             results. Default is None.
+        save_dpl : bool
+            If True, save dipole results. Default: True.
+        save_spiking : bool
+            If True, save spiking results. Default: False.
+        save_lfp : bool
+            If True, save local field potential (lfp) results. Default: False.
+        save_voltages : bool
+            If True, save voltages results. Default: False.
+        save_currents : bool
+            If True, save currents results. Default: False.
+        save_calcium : bool
+            If True, save calcium concentrations. Default: False.
```

Comment:
Add `self.clear_cache=False` to the `__init__()`, this will allow you to store simulation results in a class attribute